src/include/cpu/rpp_cpu_common.hpp

```diff
@@ -4965,11 +4965,11 @@ inline void compute_generic_bilinear_srclocs_and_interpolate(T *srcPtrChannel, R
     Rpp32f weightParams[4], bilinearCoeffs[4];
     Rpp32s srcLoc[4];
     srcLT.y = (Rpp32s) srcY;                                    // Bilinear LT point y value
-    srcLT.y = std::min(srcLT.y, roiLTRB->ltrbROI.rb.y - 1);
```

Comment:
I feel this change is needed for other augmentations like resize to match with HIP
Please check and revert this change

Reply:
reverted this change, initially was in an assumption that this fixes mismatch between HIP and HOST